packages/ndla-ui/src/FactBox/FactBox.tsx

```diff
@@ -6,23 +6,28 @@
  *
  */
 
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { ReactNode, MouseEvent } from 'react';
 import BEMHelper from 'react-bem-helper';
+//@ts-ignore
 import Button from '@ndla/button';
 
-const toggleFactBox = (event) => {
-  const button = event.target;
-  const aside = button.previousSibling.parentNode;
-  aside.classList.toggle('expanded');
+const toggleFactBox = (event: MouseEvent) => {
+  const button = event.target as HTMLButtonElement;
```

Comment:
Er ikke så farlig for meg, men jeg _tror_ det går an å gjøre sånn:
```suggestion
const toggleFactBox = (event: MouseEvent<HTMLButtonElement>) => {
  const button = event.currentTarget;
```
uten at jeg egentlig kan forskjellen på` event.target` og `event.currentTarget`